docs/hydra/guides/migrate-v2.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: migrate-v2
+title: Migrate from Hydra v1 to v2
+---
+
+Follow this guide when upgrading Ory Hydra to v2. This guide covers the essential steps that are likely to be universal. If your
+deployment requires different steps, deviate from this guide at your discretion.
+
+Should you run into problems with the upgrade, please visit the community [chat](https://slack.ory.sh/) or start a
+[discussion](https://github.com/ory/hydra/discussions).
+
+:::warning
+
+Back up your data! Applying upgrades can lead to data loss if handled incorrectly.
+
+:::
+
+:::warning
+
+This is an offline migration. You will need to take Hydra offline for the duration of the migration.
+
+:::
+
+:::warning
+
+There is no down migration from v2 to v1.
```

Comment:
The term "down migration" is not recognized by dictionaries and gets about 154,000 google results. Please use "downgrade" which is a widely used and understood term that gets about 28,400,000 google results. 

[Wikipedia for "downgrade"](https://en.wikipedia.org/wiki/Downgrade)

Reply:
I'm afraid I disagree with this suggestion. "downgrade" is not the same as "down migration", and the intended audience will understand it. The meaning of "down migration" can be understood from the common term itself.

Reply:
👍🏻 okay, I think I understand things better now. This term can stay in the doc.